Cargo.toml

```diff
@@ -65,6 +65,8 @@ incremental = false
 codegen-units = 16
 
 [patch.crates-io]
+# contrafact = { path = "/home/michael/proj/contrafact" }
```

Comment:
worth making it a relative path like the others below?

Reply:
I'll just remove this, my local path is different